addons/editor-dark-mode/addon.json

```diff
@@ -1093,6 +1097,34 @@
         "darkScrollbars": false
       }
     },
+    {
+      "name": "Oceanic Dark",
+      "id": "oceanic",
+      "description": "A material theme based on the very popular blue gray (#263238) and teal (#009688) combination",
+      "values": {
```

Comment:
The values of both Oceanic Dark presets should use the same order as the rest of the presets to make them easier to read.